app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -1,7 +1,7 @@
-module UserEncryptedEmailOverrides
+module UserEncryptedAttributeOverrides
```

Comment:
I did start down the path of a shared class or concern. As I did that I realized that so much of the code was just about different naming, and not different logic, that I could generalize the naming instead.